snowfakery/cli.py

```diff
@@ -89,14 +91,6 @@ def string_int_tuple(ctx, param, value: Tuple[str, str] = None):
     "generate_cci_mapping_file",
     type=click.Path(exists=False),
 )
-@click.option(
```

Comment:
This option didn't actually work. It was superseded by the continuation-file feature.